cmd/kubeadm/app/cmd/init.go

```diff
@@ -256,6 +263,13 @@ func (i *Init) Run(out io.Writer) error {
 		return err
 	}
 
-	fmt.Fprintf(out, initDoneMsgf, path.Join(kubeadmapi.GlobalEnvParams.KubernetesDir, kubeadmconstants.AdminKubeConfigFileName), i.cfg.Token, i.cfg.API.AdvertiseAddress, i.cfg.API.BindPort)
-	return nil
+	ctx := map[string]string{
```

Comment:
Cool, I didn't know you could pass `map[string]string` to Golang templates, thought one could only use structs to get that neat behavior.